keycloak/keycloak-admin-ui
# Users: update checkbox styling and add disabled checkboxes/row functionality to modal

## Motivation

Towards #462 and #589 

JIRA:
https://issues.redhat.com/browse/APPDUX-957
https://issues.redhat.com/browse/APPDUX-956


## Brief Description


## Verification Steps



1. Go to Users section.
2. Go to the groups tab.
3. Add groups.
4. Open the "Add groups" modal again and verify that the group you just added is not selectable.
5. Verify that the groups without subgroups do not have an angle icon at the end of the data list item and are not selectable.

## Checklist:


## Additional Notes
